[
    {
        "content": "<p>Am I reading the logs wrong, or is CI hanging unexpectedly? <a href=\"https://github.com/leanprover-community/mathlib/pull/7118/checks?check_run_id=2375062027\">https://github.com/leanprover-community/mathlib/pull/7118/checks?check_run_id=2375062027</a></p>",
        "id": 235092570,
        "sender_full_name": "Yakov Pechersky",
        "timestamp": 1618772412
    },
    {
        "content": "<p>I think IME usually that's a runner somehow transiently getting stuck (i.e. not your fault). Unless it's happening on other PRs probably would cancel the job and re-run it.</p>",
        "id": 235093630,
        "sender_full_name": "Julian Berman",
        "timestamp": 1618773414
    },
    {
        "content": "<p>I wouldn't cancel it yet.</p>",
        "id": 235095703,
        "sender_full_name": "Bryan Gin-ge Chen",
        "timestamp": 1618775546
    },
    {
        "content": "<p>The updates from GitHub actions sometimes get stuck while the job still runs in the background.</p>",
        "id": 235095735,
        "sender_full_name": "Bryan Gin-ge Chen",
        "timestamp": 1618775593
    },
    {
        "content": "<p>Yeah, when I've seen this before it's just github failing to indicate that it's progressed to the next stage</p>",
        "id": 235095977,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1618775872
    },
    {
        "content": "<p>it's ticking along nicely now, by the looks of things</p>",
        "id": 235096318,
        "sender_full_name": "Kevin Buzzard",
        "timestamp": 1618776191
    }
]